install/kubernetes/cilium/templates/hubble-cli.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.hubble.cli.enabled }}
+apiVersion: v1
+kind: Pod
```

Comment:
I wonder if it made sense to make this into a proper Deployment with a readyness probe (that e.g. invokes `hubble status`, similar to how the `connectivity-check.yaml` works.

Reply:
So we would have `replicas: 0` and `kubectl scale deployment -n kube-system --replicas 1 hubble-cli` when needed? That seems cleaner in a way than `kubectl create`'ing the Pod. Not sure about the readiness Probe as `hubble status` might just not work when we need this Pod to debug Hubble issues.

Reply:
I think it depends what the use-case of the YAML file itself should be. I would have proposed `replicas: 1` by default. The idea of a deployment together with a readiness probe is to allow it to be used as a smoke test: If the deployment itself becomes ready, then Relay works as expected.

I guess we could also use a plain pod for that, I just haven't found plain pods to be very common, because deployments allow for a bit more robust tracking of changes of changes, i.e. one can do roll backs.